src/main/java/com/lpvs/service/LPVSDetectService.java

```diff
@@ -152,12 +138,11 @@ public void runOneScan() {
                                     webhookConfig, scanResult, detectedConflicts);
                     log.info(report);
                 }
+                log.info("Single scan completed.");
             } catch (Exception ex) {
                 log.error("\n\n\n Single scan finished with errors \n\n\n");
-                log.error("Can't triger single scan " + ex);
+                log.error("Can't trigger single scan: " + ex.getMessage());
             }
-            // exiting application
-            eventPublisher.publishEvent(new ExitCodeEvent(new Object(), 0));
```

Comment:
Why do you skip exiting from the app?

Reply:
When we do sys call with System.exit(0) we have JVM exit. 
First - it causes critical flaw in application (we cannot run this through security checker)
Second - it closes JVM during runtime both on build instal and testt phases

Reply:
OK. Will be added in a separate PR.